Orange/widgets/classify/owadaboost.py

```diff
@@ -28,6 +29,11 @@ class OWAdaBoostClassification(OWBaseLearner):
 
     DEFAULT_BASE_ESTIMATOR = SklTreeLearner()
 
+    class Error(OWBaseLearner.Error):
```

Comment:
We decided it is best to have an Error message and require the user removes the invalid input before continuing.